### DISCUSSION TITLE
Problems with package installation

### DISCUSSION THREAD

**Discussion by astaaudzi** [fishsizeproject/CPUEcourse#1] [Q&A]
Hi everyone, 

this is discussion group to write about any challenges you might be having with installing packages required for the course. We invite everyone to contribute. If you have questions - post them here. If you found solutions - please also post them here, so others can benefit from it.

**Comment by solisr1005** [comment]
Hi. rtools is not available for Mac. should I use a windows computer?

**Reply by astaaudzi** [reply]
You should be able to run all the scripts on a Mac computer without having to install Rtools. Let me check (I am a windows user, but Carl uses Mac, as far as I am aware)

**Comment by MauroMardones** [comment]
If someone have any trouble with INLA install, is necessary this previously;

`if (!requireNamespace("BiocManager", quietly = TRUE))
    install.packages("BiocManager")
BiocManager::install(c("graph", "Rgraphviz"), dep=TRUE)
`

**Reply by lmusinguzi** [reply]
I am still not able to have INLA installed
![image](https://user-images.githubusercontent.com/57838270/203733349-fb66a434-8d6a-40ea-a1ff-3327fb5c2194.png)

**Reply by MauroMardones** [reply]
I think you should increase the render time in R. The default is 60 seconds. You can change previously install `inlatools`

`options(timeout=1000)`

This works for me.
